src/bg/user-script-registry.js

```diff
@@ -155,6 +155,24 @@ function onListUserScripts(message, sender, sendResponse) {
 }
 window.onListUserScripts = onListUserScripts;
 
+function onUserScriptGetByName(message, sender, sendResponse) {
+  if(!message.scriptNamespace) {
```

Comment:
Please follow existing code style, consistently.  There should always be a space after an if.  80 column maximum., probably some other things I haven't seen yet.

https://greasemonkey.github.io/style.html